general/community-standards.md

```diff
@@ -0,0 +1,52 @@
+
+## Mission Statement
+
+* Jellyfin aims to be the best media streaming platform possible, built entirely by volunteers without any monetary gain, proprietary/locked features, or unreasonable centralization.
+
+* Jellyfin is a project made up entirely and exclusively of Volunteers who donate their free time to the project.
+
+* Fostering a community of respectful and productive contributors is central to our success and longevity.
+
+* Jellyfin is not and will never be under the control of any corporation or profit-driven entity, and does not exist to make money for anyone, including any volunteer contributors or the project leadership.
+
+## Code of Conduct
+
+At all times when interacting with the Jellyfin community via any method (Matrix, Reddit, Forums, etc.), you must abide by the following:
+
+* **Respect others and remember the Human.** Do exhibit kindness and empathy to others, and make them feel welcome. Do not antagonize, flame, insult, demean, abuse, or harass others. Do not use slurs or sexualized language. Do not dox or otherwise expose others' private information, even if it is shared publicly elsewhere.
+
+* **English is the primary working language of Jellyfin, but the majority of our community do not speak English as their primary language**. Be patient when language issues arise, and do not mistake incomplete language knowledge for ignorance or worse. If you are having trouble communicating an idea in English, please post in your native language and ask for help, and someone is likely to understand. As per rule 1, abuse of community members due to language issues is not tolerated.
+
+* Jellyfin, as set out in our Mission Statement, is created by volunteers exclusively. **No volunteer contributor owes anything whatsoever to any other contributor, any user, or the project itself**. This includes time, mental focus, interaction, or any other effort. Contributors are free to come and go as they please, though the administrative team would kindly request that knowledgeable contributors inform us of long-term departures to ensure we are aware, should the need arise.
+
+* If you have questions to ask of the community, **please choose the appropriate location (see our [Getting Help](xref:getting-help) page) and ask your question in full, immediately, with as much detail as possible**. Author issues aside, [Asking Questions the Smart Way](http://www.catb.org/~esr/faqs/smart-questions.html) is a valuable resource. **Do not pester or harass** community members for help or to answer questions, **do not require others to pry information** out of you, and **do not spam questions** - they will be answered when they are answered.
+
+* Jellyfin is a media server system for your own media collection. Our communities are not a place to obtain media. **Do not engage in, encourage, or facilitate piracy**. Do not ask questions about where to obtain media. Do not ask for access to other users' servers. Do not sell or transfer access to servers. 
```

Comment:
I think the last sentence is a bit vague. As long as a party holds access to the distributed media, it should be ok if they sell access to their own managed server instances. We may of course prohibit advertisement of such in the public Jellyfin channels, but that would have to be clarified then.

Reply:
Yea that's fine and not what I'm implying with this rule, maybe "private" servers would clarify it.

Reply:
Hopefully my updated wording is a bit better.

Reply:
I'm still not 100% sure how that "transfer" is meant, but I'm not a native English speaker, so that could be a reason.

Reply:
So in this case it's implying a "quid-pro-quo" arrangement rather than outright sale. If I just say "Don't sell access", someone could try to use the loophole "but I'm not *selling* access, I'm trading an XBox for access, that's not selling". This wording attempts to prohibit that.

Reply:
Ah, now I understand, thanks for the explanation.

Reply:
No problem! I could also say "trade", but that's a bit narrower, and I like writing very legalese language.